site/content/support.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Where to Go for Support"
+draft: false
+weight: 1100
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-OOO"
+---
+
+## Support Policy
+
+NGINX Gateway Fabric adheres to the support policy detailed at [nginx.com/support](https://nginx.com/support).
+
+## Commercial Support
```

Comment:
As detailed in another thread there's no plans for a support package yet, but it seems like these two sections could be combined into one. Having commercial support paths before communal ones also has an open core vs open source feel to it, which might be an ethos choice.

Reply:
It is an ethos choice.  However, the commercial support statement should be very short and direct the customer in the proper direction.